docs/admin/authorization.md

```diff
@@ -108,12 +111,19 @@ If at least one line matches the request attributes, then the request is authori
 To permit any user to do something, write a policy with the user property unset.
 To permit an action Policy with an unset namespace applies regardless of namespace.
 
+### Kubectl
+
+Kubectl uses the `/api` endpoint of api-server to negotiate client/server versions. To validate objects send to the API by create/update operations, kubectl queries certain swagger resources. For API version `v1` those would be `/swaggerapi/api/v1` & `/swaggerapi/experimental/v1`.
+
+ When users, who are restricted by namespaces, should be able to use the kubectl client, `/api` has to be exempted from authorization checks via a `"nonResourcePath":"/api"` entry in the policy file. When those users are suppossed to create or update resources, additional policies for the swagger paths `"nonResourcePath":"/swaggerapi/api/v1"`, `"nonResourcePath":"/swaggerapi/experimental/v1"` (assuming `v1` is the API version being used) have to be added (see example below).
```

Comment:
@deads2k: currently this only applies to users restricted to namespaces, AFAIK this is intended behaviour:

    func (p policy) matches(a authorizer.Attributes) bool {
      if p.subjectMatches(a) {
        if p.Readonly == false || (p.Readonly == a.IsReadOnly()) {
          switch {
          case p.NonResourcePath != "":
            if p.NonResourcePath == a.GetNonResourcePath() {
              return true
            }
          case p.Resource == "" || (p.Resource == a.GetResource()):
            if p.Namespace == "" || (p.Namespace == a.GetNamespace()) {
              return true
            }
          }
        }
      }

      return false
    }

Given a policy `{"user": "admin"}` When the user is `admin`, the policy has no `resource` restriction and no `namespace` restriction either, the action is authorized. Should we change this?